### ISSUE TITLE
[Question]Some Metals are not in the OreDict, intentional or not yet added in?

### ISSUE THREAD

**Issue by SteelShot** [MJRLegends/ExtraPlanets#23]
The Title pretty much explains it, some of your metals like zinc, mercury, titanium ect. are not in the OreDict. just wondering if this is intentional for Progression or they are not yet added in to it?

**Comment by MJRLegends** [comment]
well i didnt think any mod used them so didnt think OreDict was needed, but i can add them to the OreDict if you want

**Comment by SteelShot** [comment]
That would be nice, there a few mods that use Zinc and Tungsten, well at least for me. And i have unidict mod installed, so also unifying the ores would work then :)

**Comment by MJRLegends** [comment]
Ok i will add it for you :P

**Comment by SteelShot** [comment]
Thank you very much

**Comment by MJRLegends** [comment]
Im guessing your wanting the ore to be to oredict?

**Comment by SteelShot** [comment]
Well, i guess the standart things the ores,ingots,blocks i think is enough, and nothing is more added in as i remember

**Comment by MJRLegends** [comment]
i've done the ore's c3ce321b2aecf1a5c95e11d765dbd1f93e04225b

**Comment by MJRLegends** [comment]
Currently none of the "ingots" or "block of" are not set to ore dict

**Comment by MJRLegends** [comment]
I will do all the "ingots" or "block of" for all of them

**Comment by SteelShot** [comment]
You the man! xD

**Comment by MJRLegends** [comment]
The update including this will be out in the next 24hr

**Comment by MJRLegends** [comment]
Ingots/gems ef4278b4066756f3c96371cbe707730e9129094e will do the "block of" blocks tomorrow and then release

**Comment by MJRLegends** [comment]
Version is out! Let me know if you get any issues :D

**Comment by SteelShot** [comment]
Not going to open a new thread, but there is still few ores left and carbon ore has too many oredict entries.

Carbon ore has the following: oreCarbon, blockCarbon, orePalladium and oreMagnesium
Jupiter Palladium, Nickel  and Red Gem ores don't have any entries.
Saturn Magnesium ore is registered as oreWhiteGem.

Everything else seems fine

**Comment by MJRLegends** [comment]
Im releasing another update in the next 24hrs so it will be fixed there

**Comment by MJRLegends** [comment]
Fixed 29f512c8638b08a4d2f9f5f5bea69843890d2895

**Comment by MJRLegends** [comment]
Update v1.0 Pre-3.6.6 is out

**Comment by janagyjr** [comment]
I know this is very old, but Titanium blocks don't seem to be oredicted still in 1.8.9. I went to adjust the recipe for the tier 2 rocket launchpad (for the tier 4-8 rockets) and had to add in a mirror recipe using another mod's titanium blocks (I have all my metals going in to compacting drawers, Storage drawers shows Extra Planets ingots but another installed mod's blocks (Another One Bites the Dust) and when going to craft the pad, I had to manually create the EP titanium blocks from the ingots. I think this is related hence posting on this issue (my apologies if it's not related and I will open a new issue).
